### DISCUSSION TITLE
New Wlan Pi Pro not powering on

### DISCUSSION THREAD

**Discussion by cni-ruiz** [WLAN-Pi/feedback#26] [Hardware - WLAN Pi Pro] [answered]
Received a new unit, but have not yet been able to power it on.   I have tried charging via POE as well as using my laptop with the USB-C option.  I show charging indicator lights when plugged in, but the charge indicator never goes up.  It is stuck just blinking on the lowest power option even after being left for hours.   I have tried pulling and putting the battery back in, but the result stays the same.

**Comment by jolla** [comment]
As discussed in other channels, as a first step please follow the steps in the troubleshooting section of the user guide.  https://userguide.wlanpi.com/hardware/wlan-pi-pro/troubleshooting

**Comment by cni-ruiz** [comment, marked as answer]
Issue ended up being a bad battery.  Once swapped out with another the unit powered on with no issues.  Following up so the symptoms of a failed battery are known.

Lights only come on when plugged into a power source.  Battery indicator only blinks at lowest level and never goes up.
